test/libsolidity/semanticTests/various/copying_bytes_multiassign.sol

```diff
@@ -0,0 +1,28 @@
+contract receiver {
+    uint public received;
+    function recv(uint x) public { received += x + 1; }
+    fallback() external { received = 0x80; }
+}
+contract sender {
+    constructor() { rec = new receiver(); }
+    fallback() external { savedData1 = savedData2 = msg.data; }
+    function forward(bool selector) public returns (bool) {
+        if (selector) { address(rec).call(savedData1); delete savedData1; }
+        else { address(rec).call(savedData2); delete savedData2; }
+        return true;
+    }
+    function val() public returns (uint) { return rec.received(); }
+    receiver rec;
+    bytes savedData1;
+    bytes savedData2;
+}
+// ====
+// compileViaYul: also
+// ----
+// val() -> 0
```

Comment:
This is missing the `recv()` call from the original test. I guess it's because you could not call it on the `receiver` contract?

@chriseth Do we have support for that? `TestFileParser` does not seem to have code for handling something like `receiver.recv()`.

Reply:
A vaguely remember that there is a feature that allows isoltest to call a non-existing function.

Reply:
Added the recv() call to the test case. Indeed, the test case fails to compile. 

Error message is : 
`Exception during test: /solidity/test/libsolidity/SemanticTest.cpp(433): Throw in function solidity::frontend::test::TestCase::TestResult solidity::frontend::test::SemanticTest::runTest(std::ostream&, const string&, bool, bool, bool)
Dynamic exception type: boost::wrapexcept<std::runtime_error>
std::exception::what: The function recv() is not known to the compiler`

Reply:
Figured that the primary role of the recv() function call was to invoke the fallback() function within the contract. So, we replaced recv() with a call to the constructor ( since the parameters are incorrect, the fallback() function will be invoked )